Tensile/Common.py

```diff
@@ -843,6 +843,10 @@ def getArchitectureName(gfxName):
     "AssertSizeLessThan":    -1,
     "AssertSizeMultiple":    -1,
 
+    # Assertions that require arithmetic intensity to be specified value.
```

Comment:
I am not sure how/when we need to use this.
Maybe better to put some explanation about the definition of arithmetic intensity here?

Reply:
Added some more comments here